docs/release-notes/february-2024.md

```diff
@@ -0,0 +1,146 @@
+---
+slug: february-2024
+hide_title: true
+description: >-
+    Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+title: February 2024
+tags:
+  - Release notes
+---
+
+# Semgrep release notes for February 2024
+
+## 🔧 OSS Engine
+
+* The following versions of the OSS Engine were released in February 2024:
+  * [<i class="fas fa-external-link fa-xs"></i>1.59.1](https://github.com/semgrep/semgrep/releases/tag/v1.59.1)
+  * [<i class="fas fa-external-link fa-xs"></i>1.60.0](https://github.com/semgrep/semgrep/releases/tag/v1.60.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.60.1](https://github.com/semgrep/semgrep/releases/tag/v1.60.1)
+  * [<i class="fas fa-external-link fa-xs"></i>1.61.1](https://github.com/semgrep/semgrep/releases/tag/v1.61.1)
+  * [<i class="fas fa-external-link fa-xs"></i>1.62.0](https://github.com/semgrep/semgrep/releases/tag/v1.62.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.63.0](https://github.com/semgrep/semgrep/releases/tag/v1.63.0)
+
+## 🌐 Cloud Platform
+
+### Added
+
+- API: Added a `rule` object under `findings` with the following fields: 
+    - `name`
+    - `message`
+    - `confidence`
+    - `category`
+    - `subcategories`
+    - `technologies`
+    - `vulnerability_classes`
+    - `cwe_names`
+    - `owasp_names` <!-- 12868 --> 
+
+
+### Changed
+
+### Fixed
+
+- Fixed a bug where the navigation sidebar covered mobile screens and could not be collapsed. <!-- 12876 -->
+
+## 💻 Code
+
+### Added
+
+* Added new rules for Elixir and the Phoenix framework, covering various security and correctness issues. These are available in the `p/elixir`
+  ruleset. <!-- Do we need to mention that this is for Pro users only? -->
+* Added support for Python, with a focus on the Flask ecosystem, to the Semgrep
+  Pro Engine.
+* Added ability to collapse and turn off patterns in a rule.
+* Added the ability to distinguish between which rules are available to Semgrep
+  Pro Engine users and which ones are available to Semgrep OSS users.
+* Added support for nested record patterns on the left-hand side of an
+  assignment.
+* `metavariable-regex` can now match on metavariables of interpolated strings
+  that use variables with known values.
+* Added support for parsing Swift Package Manager manifest files and lock files.
+* **Taint analysis**:
+  * Added support for Python constructors
+  * Added support for index sensitivity. Semgrep tracks taint on individual
```

Comment:
Half the list has full stops, and the other half has none, may I suggest using full stops?